app/views/RoomActionsView/ActionsList/RoomItem.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { withNavigation } from 'react-navigation';
+import PropTypes from 'prop-types';
+
+import Avatar from '../../../containers/Avatar';
+import Status from '../../../containers/Status';
+import DisclosureIndicator from '../../../containers/DisclosureIndicator';
+import Touch from '../../../utils/touch';
+import RoomTypeIcon from '../../../containers/RoomTypeIcon';
+
+import { COLOR_WHITE } from '../../../constants/colors';
+import styles from './styles';
+import sharedStyles from '../../Styles';
+
+const RoomItem = ({
+	name, baseUrl, user, member, route, params, testID, navigation
+}) => {
+	const { room } = params;
+	const { navigate } = navigation;
+
+	return (
+		<Touch
+			onPress={() => navigate(route, params)}

```

Comment:
Can you pass `onPress` as a prop?
You wouldn't have to create it as inline on every render that way.